package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "image-picker",
+  "version": "0.3.0",
+  "homepage": "http://rvera.github.com/image-picker",
+  "authors": [
+    "Rodrigo Vera"
+  ],
+  "description": "Image Picker is a simple jQuery plugin that transforms a select element into a more user friendly graphical interface.",
+  "main": "image-picker/image-picker.js",
```

Comment:
Good question - I just had a brief look at  how a couple of other libraries are doing it (jQuery, jQuery UI, Underscore, KnockoutJS).  These examples all referenced the unminified versions - so perhaps that would be a safe bet.

Besides, those pulling-in their dependencies via `npm` are, I would say, quite likely to be using some form of build process that often involves an `uglification` step.  In the worst case scenario, they can change what file `require('image-picker')` resolves to.

What do you think?